public/src/MockGRPCManager.ts

```diff
@@ -656,14 +655,11 @@ export class MockGrpcManager {
 
     // /* REPOSITORY */ //
 
-    public getRepositories(courseID: bigint, types: Repository_Type[]): Promise<IGrpcResponse<Repositories>> {
+    public getRepositories(courseID: bigint): Promise<IGrpcResponse<Repositories>> {
         // TODO
         if (!this.courses.courses.find(c => c.ID === courseID)) {
             return this.grpcSend<Repositories>(null, new Status({ Code: BigInt(Code.Unknown), Error: "Course not found" }))
         }
-        types.forEach(() => {
-            // TODO
-        })
         //const repos = this.repositories.getRepositoriesList().filter(r => r.courseID === courseID && types.includes(r.getType()))
```

Comment:
Perhaps this should be replaced with code that returns some dummy repositories for mocking instead of the empty repositories.

Reply:
@JosteinLindhom: could you take a look at this? Not sure if the mock method is used or needed in the near future, but if not, perhaps we can just remove the line.